src/app/file-manager/file-explorer/file-explorer.component.ts

```diff
@@ -59,7 +62,7 @@ export class FileExplorerComponent implements OnInit {
     this.cols = [
       { field: 'path', header: 'Name', footer: 'Name' },
       { field: 'size', header: 'Size', footer: 'Size' },
-      { field: 'type', header: 'Type', footer: 'Type' }
+      { field: 'updated', header: 'Modified', footer: 'Modified' }
     ];
 
     this.items = [
```

Comment:
These menu items are bound and have commands but I don't think we need View or Delete for this app. Also I wasn't able to see them by bringing up the context menu; does `p-contextMenu` work in electron?

Reply:
Context menu has been removed.